source/verus/src/main.rs

```diff
@@ -75,19 +63,270 @@ fn main() {
             }
         }
     }
+
     cmd.arg("run")
         .arg(TOOLCHAIN)
         .arg("--")
         .arg(verusroot_path.join(RUST_VERIFY_FILE_NAME))
-        .args(args)
+        .args(args_bucket)
         .stdin(std::process::Stdio::inherit());
-    match platform::exec(&mut cmd) {
+
+    if !std::env::args().any(|arg| arg.starts_with("--color")) {
+        if is_terminal::is_terminal(&std::io::stderr())
+            || is_terminal::is_terminal(&std::io::stdout())
+        {
+            cmd.arg("--color=always");
+        } else {
+            cmd.arg("--color=never");
+        }
+    }
```

Comment:
This doesn't account for if the user explicitly ran either `verus --color auto foo.rs` or `verus --color=auto foo.rs` (in which case you should remove those argument(s) and then do this manual update you are doing)